docs/localInstallation.md

```diff
@@ -464,9 +465,10 @@ If you need to generate a MD5 password, you can use:
 
 #### Initialising Users
 To start using or try Phis, two users are created automatically:
-* admin@phis.fr/admin for administrative rights
-* guest@phis.fr/guest for restricted rights
+* admin@opensilex.org/admin for administrative rights
+* guest@opensilex.org/guest for restricted rights
 See Phis user documentation for explanation and add other users.
```

Comment:
link to the user documentation: https://opensilex.github.io/phis-docs-community/

Reply:
```suggestion
See [Phis user documentation](https://opensilex.github.io/phis-docs-community/) for further explanation on users management and how to add new users.
```